.github/workflows/pr_validate.yaml

```diff
@@ -204,11 +204,6 @@ jobs:
 
           echo "::set-output name=body::$UNCHECKED_BODY"
 
-      - id: check-case-sensitivity
```

Comment:
Why was this removed?

Reply:
This is a left over debug step leftover from the PR which fixed the uppercase branch name (#291).